openassessment/xblock/test/test_studio.py

```diff
@@ -246,22 +247,17 @@ def test_check_released(self, xblock):
 
     @scenario('data/self_then_peer.xml')
     def test_render_editor_assessment_order(self, xblock):
-        # Initially, the editor assessment order should be the default
-        # (because we haven't set it yet by saving in Studio)
-        # However, the assessment order IS set when we import the problem from XML,
-        # and it differs from the default order (self->peer instead of peer->self)
         # Expect that the editor uses the order defined by the problem.
         self._assert_rendered_editor_order(xblock, [
-            'example-based-assessment',
```

Comment:
See below for fix - `example-based-assessment` should have never been appearing here, since it is not defined in the problem.